{"uuid":"cdd4a55d-07cf-4327-a067-493c558cce43","name":"Scenario Outline: Verify that user receives error for invalid python code for Max Consecutive Ones Question","children":["8381b91f-725a-4355-a0b5-ac94041c3517"],"befores":[{"name":"hooks.HooksPage.setUp()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1747069573068,"stop":1747069575392}],"afters":[{"name":"hooks.HooksPage.tearDown(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1747069578453,"stop":1747069578597}],"start":1747069573067,"stop":1747069578597}